web/includes/functions.php

```diff
@@ -27,6 +27,9 @@ function ob_get_clean() {
   }
 }
 
+require_once( 'logger.php' );
+require_once( 'database.php' );
```

Comment:
Why did you add these?  by the time anything includes functions.php it is generally understood that these will have been included already, in index.php.  Did you find that they were necessary?